Add bash remediation for rpm_verify_hashes

mildass opened the pull request:
#### Description:
Add bash remediation for Verify File Hashes with RPM rule.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/32adf7de-363e-4a74-858e-39ba446ddf68) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/32adf7de-363e-4a74-858e-39ba446ddf68) completed: **No new issues**

redhatrises commented on shared/fixes/bash/rpm_verify_hashes.sh:
```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+
+# Find which files have incorrect hash (not in /etc, because there are all system related config. files) and then get files names
+files_with_incorrect_hash="$(rpm -Va | grep -E '^..5.* /(bin|sbin|lib|lib64|usr)/' | sed -r 's;^.*\s(.+)$;\1;g' )"
```
This script should probably be similar to `shared/fixes/bash/rpm_verify_permissions.sh`

Also, we shouldn't limit to specific directories.

yuumasato replied:
Would it make sense to update OVAL definition too?
https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/checks/oval/rpm_verify_hashes.xml#L33

dahaic replied:
Note that it was deliberate change, to limit the directories in the oval. Are we running in circles?

redhatrises replied:
@dahaic no you are right. I am the one running in circles.

yuumasato commented on shared/fixes/bash/rpm_verify_hashes.sh:
```diff
@@ -0,0 +1,12 @@
+# From files names get package names and change newline to space, because rpm writes each package to new line
+packages_to_reinstall="$(rpm -qf $files_with_incorrect_hash | tr '\n' ' ')"
+
+if which dnf ; then
```
We can use jinja macros and check for `pkg_manager` and avoid this `if` in final remediation snippet.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/78e9c376-0746-401e-84a6-8828536a2515) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/78e9c376-0746-401e-84a6-8828536a2515) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/95fedb50-8456-4fea-9b03-d75f0436173f) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/95fedb50-8456-4fea-9b03-d75f0436173f) completed: **No new issues**

mildass commented:
@yuumasato Updated.
And I found out that one of tests checks hashes in all folders (but OVAL/remediation doesn't check/fix files in e.g. `/etc` folder). I changed that test to check only folders which are being checked/fixes and added as new commit to this PR.

yuumasato commented on tests/data/group_system/group_software/group_integrity/group_software-integrity/group_rpm_verification/rule_rpm_verify_hashes/fresh_system.pass.sh:
```diff
@@ -4,6 +4,6 @@
 # verify (-V) all installed (-a) packages digests,
 # if digest differs from package metadata, then attribute '5' is printed
-result=$(rpm -Va | grep '^..5')
+result=$(rpm -Va | grep '^..5.* /(bin|sbin|lib|lib64|usr)/')
```
@mildass For me, this needs `-E` for grep to list the filenames.

mildass replied:
@yuumasato Yes, sorry. I forgot it and I didn't think that's wrong, because test passed (but it passed because of empty grep output).

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/c2e6002e-ce6c-468d-a83b-df5f73b4acac) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/c2e6002e-ce6c-468d-a83b-df5f73b4acac) completed: **No new issues**

yuumasato commented:
Fail in Travis CI is due to linkchecker, unrelated to this PR.
